### ISSUE TITLE
release process stuff

### ISSUE THREAD

**Issue by steveklabnik** [azerupi/mdBook#290]
@azerupi ; I just cut 0.0.22 for @dtolnay and the cookbook; they had several fixes they wanted to get out. That's when I noticed you had it as 0.0.22-pre; now that it's not just you that can do releases, we should talk about stuff like

* how version should be in cargo.toml
* do we push tags, and if so, what format

etc

/cc @frewsxcv

**Comment by azerupi** [comment]
Yes, so actually for the last couple of releases I have been using cargo-release because it allows to quickly release a new version and not forget any of the steps. It's cargo-release that put the version number to 0.0.22-pre after the release. 

I do want to push tags when releasing because it allows to quickly find the point in the git history where a specific release was done. First I tagged them as `v0.0.x` but cargo-release tags them as `0.0.x` both are fine for me really. 

So currently versions are 0.0.x, but once we push the TOML config we go for 0.1.x and follow semver a little more closely.

**Comment by steveklabnik** [comment]
Ah neat! I am okay with whatever, personally.
